frontend/src/metabase/public/components/EmbedModal/EmbedModalContent/EmbedModalContent.tsx

```diff
@@ -20,6 +25,15 @@ import type {
 import { SelectEmbedTypePane } from "../SelectEmbedTypePane";
 import { EmbedModalContentStatusBar } from "./EmbedModalContentStatusBar";
 
+const countEmbeddingParameterOptions = (embeddingParams: EmbeddingParameters) =>
+  Object.values(embeddingParams).reduce(
+    (acc, value) => {
+      acc[value] += 1;
+      return acc;
+    },
+    { disabled: 0, locked: 0, editable: 0 } as Record<string, number>,
```

Comment:
I'm not sure if I'm right about it, but if EmbeddingParams can only have "disabled"|"locked"|"editable" as value, then we can change the type to 
```ts
export type EmbeddingParameters = {
  [key: string]: "disabled" | "locked" | "editable";
};
```
And remove the 'as Record...` here?
<img width="797" alt="image" src="https://github.com/metabase/metabase/assets/1914270/4bd166ae-d633-454e-a0cd-f78fc630fe31">

Reply:
Also we could use [`_.countBy`](https://underscorejs.org/#countBy)